lib/nuget-parser/parsers/dotnet-core-v2-parser.ts

```diff
@@ -110,11 +110,31 @@ function getRestoredProjectName(
   );
 }
 
+export function extractLocalPackages(libs: Record<string, any>): string[] {
+  const localPackages: string[] = [];
+
+  for (const [key, value] of Object.entries(libs)) {
+    if (!key.includes('runtimepack')) {
+      if (!value.serviceable && !value.sha512) {
+        localPackages.push(key);
+      }
+    }
+  }
+
+  return localPackages;
+}
+
+function versionStringToInt(version: string) {
```

Comment:
Maybe a comment with an example. Why is it needed? This looks like something that's going to fail when MS starts to do stuff differently. Is there a way around it?

Reply:
removed the method: not needed anymore